### ISSUE TITLE
Value removes from field after clicking on it

### ISSUE THREAD

**Issue by tvdruenen** [grafana/grafana#12442]
### What Grafana version are you using?
4.6

### What datasource are you using?
Elasticsearch (5.6)

### What OS are you running grafana on?
Docker

### What did you do?
Trying to click on a field name that is filled in to copy it or check which one it is. For instance after "Group By Terms". If I click on this it wil disappear and a dropdown list will be shown. Now I can't see which one it was. Now it is impossible to find out which field is exactly selected.

### What was the expected result?
The dropdown menu shows (as it does) but also the field that is filled in would stay there so you can copy/change/select/view it.

### What happened instead?
The fieldname disappears. 

![middleware grafana issue 1](https://user-images.githubusercontent.com/6881822/42030496-48017ea0-7ad3-11e8-9351-086386d26608.png)

**Comment by marefr** [comment]
Adding influxdb. This is probably a problem for all datasources using the typeahead component in query editors.

**Comment by torkelo** [comment]
this is not a bug but more a UX issue as it done by design to in order to provide the typeahead/dropdown menu. If we did not remove it you would see no other options in the dropdown.

**Comment by marefr** [comment]
@torkelo yes I totally understand that. If I'm not incorrect the graphite query editor's drop downs doesn't clear the value. 

I discussed this problem with a guy who uses influx so I thought we add influx to this issue as well.

**Comment by aocenas** [comment]
@dprokop you may be working on a new input so this may come into consideration.
